metricbeat/docs/fields.asciidoc

```diff
@@ -2390,6 +2391,47 @@ type: integer
 The average queue time in ms over the last 1024 requests.
 
 
+[[exported-fields-jolokia]]
+== Jolokia Fields
+
+Jolokia Module
+
+
+
+[float]
+== jolokia Fields
+
+
+
+
+[float]
+== dynamic Fields
+
+dynamic
+
+
+[float]
+=== jolokia.dynamic._namespace
```

Comment:
No need to document this one. This field will be removed and placed under `metricset.namespace`.